CONTRIBUTOR_LADDER.md

```diff
@@ -1,20 +1,27 @@
 # OSM Contributor Ladder
 
-Hello! We are excited to have you contribute to Open Service Mesh (OSM)! This contributor ladder outlines the different contributor roles within the project, along with the responsibilites and privileges that come with them. Community members generally start at the first levels of the "ladder" and advance up it as their involvement in the project grows. Our project members are happy to help you advance along the contributor ladder.
+Hello! We are excited to have you contribute to Open Service Mesh (OSM)! This contributor ladder outlines the different contributor roles within the project, along with the responsibilities and privileges that come with them. Community members generally start at the first levels of the "ladder" and advance up as their involvement in the project grows. Our project members are happy to help you advance along the contributor ladder.
 
-Each of the contributor roles below is organized into lists of three types of things.
-* "Responsibilities" are things that a contributor is expected to do
-* "Requirements" are qualifications a contributor needs to meet to be in that role
-* "Privileges" are things a contributor on that level is entitled to
+Each of the roles is organized into the following categories:
+* "Responsibilities" list what the role is expected to do
+* "Requirements" are qualifications the role needs to meet
+* "Privileges" list what the role is entitled to
 
+| Role | Responsibilities | Requirements | Privileges |
+| -----| ---------------- | ------------ | -------|
+| [Maintainer](#maintainer) | Active member of the community |<ul><li>Reviewer for ≥3 months</li><li> Demonstrates deep understanding of multiple areas of the project</li><li>Exercise good judgement for the project</li><li>Maintain contributions to the project</li></ul> | <ul><li>Has GitHub or CI/CD rights to approve any pull requests</li><li>Represent the project as a Maintainer</li><li>Communicate with CNCF on behalf of the project</li></ul> |
+| [Reviewer](#reviewer) | Review contributions from other members | <ul><li>Contributor for ≥3 months</li><li>Demonstrates knowledge in an area of the project</li><li>Reviews pull requests</li></ul> | Has GitHub or CI/CD rights to approve pull requests in specific directories |
```

Comment:
@shashankram I agree with that. The goal right now is to incentivize reviews and I think reducing the complexity of the change (not adding another role) and streamlining the process to become a Maintainer will help with that.

@shalier what are your thoughts on this? I think we should describe the process of adding maintainers as codeowners for certain subdirectories and define the process of how to gain more ownership. We'll also need to update the maintainer role to reflect these changes. 

I think we can revisit adding more contributor roles as the OSM project grows.

Reply:
@shalier yeah, let's hold off

Reply:
Hi, I would like to comment on a new role, currently reviewing PRs takes more time as fewer maintainers are there for the project. By adding a reviewer role, it will be easier for a new contributor to come up to the maintainer role as the path will be more clear and more transparent. Also, it will help in faster reviewing.

Reply:
@mudit-01 I think the proposed changes to the maintainer role will make it easier to become a maintainer since ownership doesn't need to be over the entire project. A contributor can be promoted to a maintainer when they demonstrate the requirements of a maintainer in at least one project area. Hopefully, the review process will be faster with more maintainers!